.circleci/config.yml

```diff
@@ -0,0 +1,20 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/python:3.6
+    steps:
+      - run: mkdir -p docs/_build/html
+      - run: wget http://sunpy.org/index.html -O docs/_build/html/index.html
```

Comment:
we need to change this

Reply:
technically you *don't* as it's just a check to see if the artifact works, it's just any html page you like that it then reports as a status to this repo. If you don't want to test this blueprint on this repo then you can delete the whole file.

Reply:
This assumes that `http://sunpy.org/index.html` is going to be up indefinitely, which is out of the bot's control. Might be better to get https://astropy.stsci.edu/data/index.html ?

Reply:
or even better http://www.astropy.org/index.html

Reply:
*takes offence and leaves* :wink:

Reply:
Offence unintended! But just that the bot is more closely associated with astropy core currently. I like the sun!